pkg/builder/common_test.go

```diff
@@ -720,6 +720,49 @@ func TestApplyReplacements(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "replacement in volumes",
+			args: args{
+				build: &v1alpha1.Build{
+					Spec: v1alpha1.BuildSpec{
+						Volumes: []corev1.Volume{
+							{
```

Comment:
nit: You can save two lines of vertical space and a tab-width of horizontal space by collapsing some braces:

```
Volumes: []corev1.Volume{{
  Name: "${name}",
  VolumeSource: corev1.VolumeSource{
    ...
  },
}},